realm-transformer/build.gradle

```diff
@@ -1,14 +1,18 @@
 buildscript {
+    ext.kotlin_version = '1.0.1-2'
```

Comment:
I would love to change the groovy as well. Those two transformers share some common functions which can be refactored. Anyone loves groovy more? @emanuelez ?